rest/api.go

```diff
@@ -298,6 +302,7 @@ func (h *handler) handleProfiling() error {
 	if isCPUProfile {
 		base.Infof(base.KeyAll, "Starting CPU profile to %s ...", base.UD(params.File))
 		err = pprof.StartCPUProfile(f)
+		h.server.pprofFileHandler = f
```

Comment:
If StartCPUProfile returns err!=nil, we need to return that error and not set pprofFileHandler.

Reply:
Fixed in latest revision